strapi/strapi
# Fixes server restarting when modifying admin files

Signed-off-by: soupette <cyril@strapi.io>



### What does it do?

It applies the correct patterns in order to prevent the server from watching front end files updates.


### Why is it needed?

In order to improve the experience when developing plugins.

### How to test it?

1. Launch the getstarted app with the `--watch-admin` file. 
2. Make some modifications to one of the `./src/plugins/myplugin/admin/src/**/*.js` file.
3. Observe that the server is not restarting.

An other method is to : 
- Generate a Strapi app with `yarn create strapi-app my-app --quickstart --no-run`
- Apply the modification to the `node_modules/@strapi/strapi/lib/commands/develop`
- Generate a plugin
- Make some modification to one of the admin file
- Observe that the server is not restarting.

### Related issue(s)/PR(s)

Fixes #11645
